src/site/stages/build/process-cms-exports/transformers/node-press_releases_listing.js

```diff
@@ -4,6 +4,23 @@ const {
   utcToEpochTime,
 } = require('./helpers');
 
+const reverseFields = reverseFieldList => ({
```

Comment:
Since `reverseFields` is only being used in one place, I think we can place this directly in the `reverseFieldListingNode` property.

Reply:
I like it like this since it keeps it clean and easier to read

Reply:
I think that's fine also then!